projeto-astromatch/projeto-astromatch/src/App.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import styled from "styled-components"
+import Header from './components/Header'
+import Perfil from './components/Perfil'
+import NP2 from './components/NP2'
+import Botoes from './components/Botoes'
+import TelaMatches from './components/TelaMatches'
+
+
+const AppContainer = styled.div`
+       
+`
+
+class App extends React.Component {
+
+  state = {
+    currentPage: "Perfil"
+  }
+
+changePage = (currentPage) => {
+  this.setState({currentPage: currentPage})
+}
```

Comment:
Como sao só duas páginas, seria mais fácil se voce usasse true ou false. Por exemplo: 
```suggestion
  state = {
   chosePersonPage: true
  }

changePage = () => {
  this.setState({currentPage: !currentPage})
}
```